src/cargo/util/paths.rs

```diff
@@ -87,6 +89,34 @@ pub fn write(path: &Path, contents: &[u8]) -> CargoResult<()> {
     })
 }
 
+pub fn append(path: &Path, contents: &[u8]) -> CargoResult<()> {
+    (|| -> CargoResult<()> {
+        let mut f = try!(
+                    OpenOptions::new()
```

Comment:
This indentation seems a bit off, the `OpenOptions` should come right after the `(` of the `try!(`